src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+/**
+ *  A class which encapsulates the user interactions between Duke and the user.
+ *  @author  Chen Guanzhou
+ *  @version v1
+ */
+public class Ui {
+
+    /**
+     * A method for Duke to tell the user how many tasks they currently have.
+     */
+    public static String taskNumberMessage(TaskList taskList) {
```

Comment:
Perhaps the method name can be getTaskNumberMessage()?

Reply:
Good point!